uptane-standard.md

```diff
@@ -780,15 +787,19 @@ The ECU SHALL verify that the latest image matches the latest metadata as follow
 
 If the ECU has secondary storage, the checks SHOULD be performed on the image in secondary storage, before it is installed.
 
-If any step fails, the ECU SHALL jump to the fifth step ({{create_version_report}}). If a step fails and the ECU does not have secondary storage, and the ECU has created a backup of its previous working image, the ECU SHOULD now install the backup image.
+If any step fails, the ECU SHALL jump to the ({{create_version_report}}) step. If an ECU does not have secondary storage, then before attempting verification, the ECU SHALL ensure a backup of a previous working image and associated metadata is created. This MAY be stored on the elsewhere within the vehicle (e.g., associated primary). Thus, if a step fails and the ECU does not have secondary storage, the ECU SHALL install the backup image to ensure it is in a working condition.
```

Comment:
This is actually a bit confusing (and was before this change). This verification is done before installation, so why would we need to re-install the backup image if we haven't installed the bogus one yet? I suppose the preceding line does say "SHOULD be performed ... before it is installed" so there is a possibility it was installed before verification.

Reply:
Thanks for the feedback, Patrick.

I agree my PR could be revised to be more cohesive with the rest of the section. (please see updated version)
The specific use case I am addressing is *pv-secondaries with no additional storage*.
This means in Step 1 of {#verify_image}, they have *already overwritten* the Targets metadata they previously contained.
Hence, any failure during these steps indicate they have overwritten their previously valid metadata (and potentially image).

I have pushed a new change to address this confusion.
Please let me know what you think.

Reply:
Looks better to me, thanks!